### PR TITLE
Add dialogue between seller and ZZB

### PR DESCRIPTION
**Pull request by WayneYuq**
Please review this, @casey-collab

### CODE REVIEW

File: Chinese/Yifan/modernCity.md [hunk 1]
```diff
@@ -6,3 +6,5 @@
 请问大哥：
 
 
+
+[盘丝洞怎么走？](../sell_film/sell_film.md)
```
Comment (casey-collab) [inline Chinese/Yifan/modernCity.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: Chinese/Yifan/sell_film/sell_film.md [hunk 2]
```diff
@@ -0,0 +1 @@
+大哥：你们这是在拍穿越剧么？你们剧组团购贴膜我给你们八折优惠！
```
Comment (casey-collab) [inline Chinese/Yifan/sell_film/sell_film.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by WayneYuq** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :sunglasses:

**Pull request merged**